pkg/volume/empty_dir/empty_dir.go

```diff
@@ -159,8 +163,13 @@ type mountDetector interface {
 type storageMedium int
 
 const (
-	mediumUnknown storageMedium = 0 // assume anything we don't explicitly handle is this
-	mediumMemory  storageMedium = 1 // memory (e.g. tmpfs on linux)
+	mediumUnknown   storageMedium = 0 // assume anything we don't explicitly handle is this
+	mediumMemory    storageMedium = 1 // memory (e.g. tmpfs on linux)
+	mediumHugepages storageMedium = 2 // hugepages
+
+	// TODO: reuse this const from "pkg/api/types.go" when PR #50859 got merged
```

Comment:
rebase against https://github.com/kubernetes/kubernetes/pull/50859 as it looks like its stabilized.

Reply:
nit: this was missed.